calc.py

```diff
@@ -25,4 +25,24 @@ def power(_a: int, _b: int) -> int:
 print(f'5 * 9 = {multiply(5, 8)}')
 print(f'42 / 7 = {divide(42, 7)}')
 print(f'3 ^ 2 = {power(3, 2)}')
-print('Everything is OK, right?')
\ No newline at end of file
+print('Everything is OK, right?')
+print()
+
+print('Now I will calculate on difrent numbers, and..')
+print('..its gonna be numbers with decimals!')
+print('15.67 + 6.789 = ', add(_a=15.67, _b=6.789 ))
+print('4.12 - 10.555 = ', subtract(_a=4.12, _b=10.555))
+print('2.5 * 3.8 = ', multiply(_a=2.5, _b=3.8))
+print('100.23 / 3.12 =', divide(_a=100.23, _b=3.12))
```

Comment:
Why did you assign values to variables?  I mean:
Why `add(_a=15.67, _b=6.789 )` and not just `add(15.67, 6.789 )`